outputs.tf

```diff
@@ -61,3 +61,11 @@ output "igw_id" {
 output "default_network_acl_id" {
   value = "${aws_vpc.mod.default_network_acl_id}"
 }
+
+output "vpc_s3_endpoint_id" {
+  value = "aws_vpc_endpoint.s3-ep.id"
+}
+
+output "vpc_dynamodb_endpoint_id" {
```

Comment:
Please add missing outputs to README. There are some of the outputs missing in that file.

Reply:
added outputs to documentation